src/shared/ui/MovieTitle/MovieTitle.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './MovieTitle.css';
+
+export const MovieTitle = ({ imageUrl, movieName, releaseYear, genres = [], onSelect }) => {
```

Comment:
let's rename to more appropriate name 
```suggestion
export const MovieTitle = ({ imageUrl, movieName, releaseYear, genres = [], onClick }) => {
```

Reply:
changed